examples/todo/src/main.rs

```diff
@@ -31,12 +31,18 @@ pub struct DbConn(SqliteConnection);
 struct Context<'a, 'b>{ msg: Option<(&'a str, &'b str)>, tasks: Vec<Task> }
```

Comment:
Since we're in this file already, I'm pretty sure this can just be `Context<'a>`. That we only have one lifetime to name, simplifying a lot of the function signatures.